app/containers/markdown/index.tsx

```diff
@@ -153,7 +144,7 @@ class Markdown extends PureComponent<IMarkdownProps, any> {
 			renderParagraphsInLists: true
 		});
 
-	get isNewMarkdown(): boolean {
+	get isNewMarkdown(): boolean | undefined {
```

Comment:
Can we make sure to return boolean?
Maybe making sure `enableMessageParser` is boolean?